sql/30-data-modeling-hralit/20-hra_journal_entities_C.sql

```diff
@@ -29,7 +29,7 @@ AS
           WHERE c.journal_issnlinking IS NOT NULL
   ORDER BY 1
         )
- SELECT DISTINCT 'https://perl.humanatlas.io/graph/hra-lit/v0.6#'::text || a.journal_nlmuniqueid::text AS "@id",
+ SELECT DISTINCT 'https://purl.humanatlas.io/graph/hra-lit/v0.6#'::text || a.journal_nlmuniqueid::text AS "@id",
```

Comment:
We don't need to add version to identifiers. `https://purl.humanatlas.io/graph/hra-lit/v0.6#` => `https://purl.humanatlas.io/graph/hra-lit#` . This is standard practice for our graphs.